tests/js/spec/components/dataExport.spec.jsx

```diff
@@ -1,154 +1,178 @@
-import {mountWithTheme} from 'sentry-test/enzyme';
+import {act, fireEvent, screen, waitFor} from '@testing-library/react';
+
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
 
 import {addErrorMessage} from 'sentry/actionCreators/indicator';
-import Button from 'sentry/components/button';
 import WrappedDataExport from 'sentry/components/dataExport';
 
 jest.mock('sentry/actionCreators/indicator');
 
+const mockUnauthorizedOrg = TestStubs.Organization({
+  features: [],
+});
+const mockAuthorizedOrg = TestStubs.Organization({
+  features: ['discover-query'],
+});
+const mockPayload = {
+  queryType: 'Issues-by-Tag',
+  queryInfo: {project_id: '1', group_id: '1027', key: 'user'},
+};
+const mockRouterContext = mockOrganization =>
+  TestStubs.routerContext([
+    {
+      organization: mockOrganization,
+    },
+  ]);
+
 describe('DataExport', function () {
-  const mockUnauthorizedOrg = TestStubs.Organization({
-    features: [],
+  beforeAll(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => void 0);
   });
-  const mockAuthorizedOrg = TestStubs.Organization({
-    features: ['discover-query'],
+  afterAll(() => {
+    // eslint-disable-next-line no-console
+    console.error.mockRestore();
```

Comment:
Not sure what this line does but maybe this was used while writing the tests? Probably fine to remove, right?

Reply:
Ah yes, I was importing DataExport directly and it was using console.error so I silenced it.

Reply:
Btw, so note on this - imo we shouldnt add feature flags inside components themselves and we should instead add this to the views. This makes it more cumbersome to test the component, because now we cant just pass an org as a prop, we need to add it to the router context which means we end up testing our feature flag component as well (which is probably not the goal)